website/cue/reference/components/sources/datadog_agent.cue

```diff
@@ -12,11 +12,12 @@ components: sources: datadog_agent: {
 		To send logs from a Datadog Agent to this source, the [Datadog Agent](\(urls.datadog_agent_doc)) configuration
 		must be updated to use:
 
-		```
+		```toml
 		logs_config.dd_url = "<VECTOR_HOST>:<SOURCE_PORT>"
 		logs_config.use_v2_api = false # source does not yet support new v2 API
 		logs_config.use_http = true # this source only supports HTTP/HTTPS
 		logs_config.logs_no_ssl = true|false # should match source SSL configuration.
+		```
```

Comment:
🤔 I thought dd agent was configured with yaml

Reply:
Hah, yes indeed, the config here is TOML, but it should be YAML. I'll update.

Reply:
WOMP